jcli/src/jcli_app/vote/mod.rs

```diff
@@ -42,12 +44,18 @@ pub enum Error {
     EncryptedTallyRead,
     #[error("failed to read decryption key bytes")]
     DecryptionKeyRead,
-    #[error("failed to read share bytes")]
-    DecryptionShareRead,
+    #[error("expected encrypted private tally, found {found:?}")]
```

Comment:
As a general rule, error messages should not use `Debug` impls. Also including the whole `Tally` enum into the error seems wasteful, especially as its longest variant should never occur here. Maybe cut the `found` type down to what can be displayed in a simple way, use an expression to conditionally format the message, or restructure the cases for "expected encrypted tally, found none" and "expected encrypted tally, found public tally".